cli/src/error.rs

```diff
@@ -89,6 +106,10 @@ impl From<serde_json::Error> for Error {
 
 impl From<io::Error> for Error {
     fn from(error: io::Error) -> Self {
+        match error.raw_os_error() {
+            Some(32) => return Error::new_ignored(), // Broken pipe
```

Comment:
Will it work to compare the error's [`kind`](https://doc.rust-lang.org/std/io/struct.Error.html#method.kind) here, and compare it to `ErrorKind::BrokenPipe`?

Reply:
It doesn't work directly but it's possible to use the error kind with construction like
```rust
x if x.kind() == ErrorKind::BrokenPipe => return Error::new_ignored(),
```
I've updated the PR.

Reply:
I think you don't even need a `match` statement:

```rust
if error.kind() == ErrorKind::BrokenPipe {
    Error::new_ignored()
} else {
    Error::new(error.to_string())
}
```

Reply:
I've fixed this in other small PR: https://github.com/tree-sitter/tree-sitter/pull/1150